doc/using/configuration.chapter.md

```diff
@@ -189,6 +189,13 @@ The following attributes can be passed in [`config`](#chap-packageconfig).
 <include xmlns="http://www.w3.org/2001/XInclude" href="../doc-support/result/config-options.docbook.xml"/>
 ```
 
+## Calling Nixpkgs from the Module System {#sec-invoke-nixpkgs-by-module}
+
+For [module system](#module-system) application authors, Nixpkgs provides a reusable module for the purpose of invoking Nixpkgs. Most users should not have to be aware of this. The documentation for its options is meant to be re-exposed in the application's documentation and is therefore not repeated here.
```

Comment:
By "module system application" you mean things like Home Manager? I'm not sure what you mean here.

Reply:
That's what I meant.
I don't know how to say this without going on a tangent. I think going on a tangent to explain this would distract from the content. Most users will know to skip the clause or skip the paragraph.

Reply:
```suggestion
For authors of applications based on the [module system](#module-system), Nixpkgs provides a reusable module for the purpose of invoking Nixpkgs. Most users should not have to be aware of this. The documentation for its options is meant to be re-exposed in the application's documentation and is therefore not repeated here.
```